src/components/components.js

```diff
@@ -7,4 +7,11 @@ export default angular.module('myModule.components', [])
   .component('myMap', myMap)
   .component('myChart', myChart)
   .component('myTable', myTable)
-  .component('myToolbar', myToolbar);
+  .component('myToolbar', myToolbar)
+  .config(WidgetsProvider => {
+    'ngInject';
+
+    WidgetsProvider.register('myMap', myMap);
+    WidgetsProvider.register('myChart', myChart);
+    WidgetsProvider.register('myTable', myTable);
```

Comment:
lol, toolbars are toolbars and widgets are widgets, they are not interchangeable no more after introduction of grid system. I see your point though, we'll allow the user to search for toolbars specifically a bit later.